network/handlers/drain.go

```diff
@@ -38,6 +38,8 @@ func (s *sysTimer) tickChan() <-chan time.Time {
 	return s.Timer.C
 }
 
+type CustomizeChecker func(w http.ResponseWriter, req *http.Request)
```

Comment:
`CheckOption` is a more classical name in our codebase.

Reply:
done